recipes/bitbucket

```diff
@@ -0,0 +1 @@
+(bitbucket :fetcher github :repo "tjaartvdwalt/bitbucket.el" :files ("bitbucket*.el"))
```

Comment:
You need not to specify `:files` attribute, it is enough to use default value.
Please see https://github.com/melpa/melpa#recipe-format